HW_4_5_Chernysh/HW4_3_Chernysh/Program.cs

```diff
@@ -0,0 +1,74 @@
+﻿using HW4_3_Chernysh.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.SqlServer.Query.Internal;
+using System.Security.Cryptography.X509Certificates;
+using System.Transactions;
+
+namespace HW4_3_Chernysh
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            using (var dbContext = new DataBaseContext())
+            {
+                //1
+
+                var first = from employeeprojects in dbContext.EmployeeProjects
+                                   join employee in dbContext.Employees
+                                   on employeeprojects.EmployeeId equals employee.EmployeeId into temp
+                                   from result in temp.DefaultIfEmpty()
+
+                                   join project in dbContext.Projects
+                                   on employeeprojects.ProjectId equals project.ProjectId into tempSecond
+                                   from resultSecond in tempSecond.DefaultIfEmpty()
+                                   select new { employeeprojects.EmployeeId, result.Title, resultSecond.Name };
+
+                //2
+
+                var second = dbContext.Employees.Select(x => new { x.EmployeeId, Difference = EF.Functions.DateDiffDay(x.HiredDate, DateTime.Now) });
+                dbContext.SaveChanges();
+
+                //3
+
+                var transaction = dbContext.Database.BeginTransaction();
+
+                try
+                {
+                    (dbContext.Clients.ToList())[3].Email = "newemail@gmail.com";
```

Comment:
you loaded the full table just to get 1 object from it? Why don't you use First/Single/etc? There are so many questions in this way to update the entity:)